addons/website/static/src/js/widgets/theme.js

```diff
@@ -362,7 +363,11 @@ var ThemeCustomizeDialog = Dialog.extend({
                             $final = $option;
                         }
                         $final.attr('data-depends', $item.data('depends'));
-                        $container.append($final);
```

Comment:
This code should not change, just give the right `$container` in the SELECTION part.